tests/network/bindingplugin_passt.go

```diff
@@ -111,20 +111,59 @@ var _ = Describe(SIG(" VirtualMachineInstance with passt network binding plugin"
 
 	Context("should allow regular network connection", func() {
 		Context("should have client server connectivity", func() {
-			Context("TCP", func() {
-				DescribeTable("Client server connectivity", func(ports []v1.Port, tcpPort int, ipFamily k8sv1.IPFamily) {
+			Context("TCP without port specification", func() {
+				var clientVMI *v1.VirtualMachineInstance
+				var serverVMI *v1.VirtualMachineInstance
+
+				const highTCPPort = 8080
+
+				BeforeEach(func() {
+					clientVMI = libvmifact.NewAlpineWithTestTooling(
+						libvmi.WithPasstInterfaceWithPort(),
+						libvmi.WithNetwork(v1.DefaultPodNetwork()),
+					)
+					clientVMI, err = kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), clientVMI, metav1.CreateOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					serverVMI = libvmifact.NewAlpineWithTestTooling(
+						libvmi.WithInterface(libvmi.InterfaceWithPasstBindingPlugin()),
+						libvmi.WithNetwork(v1.DefaultPodNetwork()),
+					)
+					serverVMI, err = kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), serverVMI, metav1.CreateOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					waitUntilVMIsReady(console.LoginToAlpine, clientVMI, serverVMI)
+
+					vmnetserver.StartTCPServer(serverVMI, highTCPPort, console.LoginToAlpine)
+				})
+				DescribeTable("Client server connectivity", func(ipFamily k8sv1.IPFamily) {
 					libnet.SkipWhenClusterNotSupportIPFamily(ipFamily)
```

Comment:
IIUC, on a single stack environment, the setup will be executed for the unsupported IP family, and then the test body will call the programmatic skip.

This has the potential to waste a few minutes times the number of tests but we can handle it in a follow-up.